### PR TITLE
MPL-33 Prevent FlowInterruptedException from being wrapped

### LINKED ISSUE #33: FlowInterruptedException should not be wrapped
Currently, all exceptions which occur during module execution are wrapped inside a MPLModuleException. Unfortunately, this also occurs for `org.jenkinsci.plugins.workflow.steps.FlowInterruptedException` which is used by Jenkins to abort a running build (e. g. manually aborted by the user or timeouts). Since the exception which is caught in the end by Jenkins is of a different type as expected, Jenkins interprets it as a build failure. 

An excerpt of a current log file with an timeout occuring during build:

> [Pipeline] End of Pipeline
> com.griddynamics.devops.mpl.MPLModuleException: Found error during execution of the module 'library:MDSD.tools Lib/resources/tools/mdsd/devops/pipeline/stages/impl/constraintBuild/modules/Build/Build.groovy':
> org.jenkinsci.plugins.workflow.steps.FlowInterruptedException
> Finished: FAILURE

when the exception is not wrapped the log looks like:

> [Pipeline] End of Pipeline
> Timeout has been exceeded
> Finished: ABORTED`

### PR DESCRIPTION
**Pull request by sdkrach**
fixes #33.

### CODE REVIEW

File: vars/MPLModule.groovy [hunk 1]
```diff
@@ -70,6 +70,13 @@ def call(String name = env.STAGE_NAME, Map cfg = null) {
     Helper.runModule(module_src, module_path, [CFG: Helper.flatten(cfg)])
   }
   catch( ex ) {
+    
+    if (ex instanceof org.jenkinsci.plugins.workflow.steps.FlowInterruptedException) {
```
Comment (sparshev) [inline vars/MPLModule.groovy]: Why don't use specialized catch for that instead of if?

Comment (sdkrach) [reply vars/MPLModule.groovy]: Didn't think of it (I am not an expert in groovy). You are right.

File: vars/MPLModule.groovy [hunk 2]
```diff
@@ -69,6 +70,11 @@ def call(String name = env.STAGE_NAME, Map cfg = null) {
     MPLManager.instance.pushActiveModule(module_path)
     Helper.runModule(module_src, module_path, [CFG: Helper.flatten(cfg)])
   }
+  catch( FlowInterruptedException ex ) {
+    //The exception is used by Jenkins to abort a running build and consequently
```
Comment (sparshev) [inline vars/MPLModule.groovy]: Could you please also add a space between '//' and the actual comment - it's hard to read when they are too close.

**Comment by sparshev** [comment]
@sdkrach thank you for the changes) Looks good.

**Pull request merged**